src/core/config/type-cast/type-casters/dom-node.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * @prettier
+ */
+const domNodeTypeCaster = (value) =>
+  value === null || value === "null" ? null : value
```

Comment:
I saw that perhaps we could check for this with
```js
value instanceof Element
```
but I wasn't 100% sure about it, so I'm returning the passed value here, just to be safe.

Reply:
It would fail for server side rendering in Node.js so it's best not to assert here at all. The code is fine.